packages/twenty-server/src/modules/contact-creation-manager/utils/filter-out-contacts-from-company-or-workspace.util.ts

```diff
@@ -21,9 +21,14 @@ export function filterOutSelfAndContactsFromCompanyOrWorkspace(
     new Map<string, boolean>(),
   );
 
+  // TODO : find a better way to enable organization1.onmicrosoft.com to add user from organization2.onmicrosoft.com
+  const isDifferentDomain = (contact: Contact, selfDomainName: string) =>
```

Comment:
I think this should have been tackled with `isWorkEmail` function. We should already filter out non-work emails here, that not the case? .onmicrosoft.com is already part of the non-work emails list

Reply:
true. as discuss I modified so that we keep the old logic, but we also add a new condition : if your selfDomain is not a workemail, we allow contact creation